packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-settings.tsx

```diff
@@ -18,4 +18,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'settings',
```

Comment:
We might want to consider renaming this to 'cog'. We want to start moving to a generic name icons that match what the icon looks like and not use-specific naming that we currently do in Teams.

For example in Font Awesome it is called 'cog' - https://fontawesome.com/icons?d=gallery&q=cog